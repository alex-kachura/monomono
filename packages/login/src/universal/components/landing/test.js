import React from 'react';
import { LandingPage } from './';

describe('[Component: LandingPage]', () => {
  let component;

  const mockProps = {
    getLocalePhrase: (key) => key,
    rootPath: '/mock/math',
  };

  beforeEach(() => {
    component = global.contextualShallow(<LandingPage {...mockProps} />);
  });

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });
});
