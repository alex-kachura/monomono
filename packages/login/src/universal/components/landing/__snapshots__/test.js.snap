// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component: LandingPage] should render correctly 1`] = `
<SideEffect(DocumentTitle)
  title="pages.landing.title"
>
  <div>
    <PageTitle
      margin={true}
      tagName="h1"
    >
      pages.landing.title
    </PageTitle>
    <Styled(WithProps(Link))
      altLink={[Function]}
      href="/mock/math/edit"
      variant="textButton"
    >
      pages.landing.sample-form-btn
    </Styled(WithProps(Link))>
  </div>
</SideEffect(DocumentTitle)>
`;
